### ISSUE TITLE
Missing tasks not in daily notes

### ISSUE THREAD

**Issue by juandpinto** [702573N/Obsidian-Tasks-Timeline#43]
Hello. Tasks that I have in notes not in my daily notes folder don't appear, even without any options or anything set. However, they do seem to appear when I add a due date to them. Is this by design? How can I make all unplanned tasks appear?

**Comment by mayukhc** [comment]
I have this same issue, tasks without a date won't appear. I think that's by design though, as per [#27 ](https://github.com/702573N/Obsidian-Tasks-Timeline/issues/27) unplanned tasks inside Inbox & daily notes will be listed inside today, so I'm guessing unplanned tasks outside those two are being ignored.

**Comment by LucaButera** [comment]
I also would like this to be an option, at least for uncompleted tasks I would like to see all of them, even if they don't have a due/scheduled/start date.

**Comment by eiffair** [comment]
Yes this is very annoying since I create tasks in different notes as I am typing them (for example in meeting Notes) and they don't show up.

**Comment by itacdonev** [comment]
This is a reason why I can't use this timeline. Lots of tasks are unplanned and not be able to see them is not an option for me, especially since they are scattered all over my vault.

**Comment by jontstaz** [comment]
I know it's a bit late but just add this inside the `getMeta()` function underneath the `for (i=0;i<tasks.length;i++) {` line:
```js
// Include tasks without date
if (!Object.values(happens).length) {
	tasks[i].order = taskOrder.indexOf("unplanned");
	timelineDates.push(today);
	happens["unplanned"] = today;
}
```

**Comment by socialchap** [comment]
> I know it's a bit late but just add this inside the `getMeta()` function underneath the `for (i=0;i<tasks.length;i++) {` line:
> 
> ```js
> // Include tasks without date
> if (!Object.values(happens).length) {
> 	tasks[i].order = taskOrder.indexOf("unplanned");
> 	timelineDates.push(today);
> 	happens["unplanned"] = today;
> }
> ```

Thanks for this. I think I'put it in the correct place, but while this does then show all 'unplanned' task, it also is showing all completed tasks here, plus isn’t then showing any tasks if I click on the Overdue filter...

Any ideas?

**Comment by torstenkoepke** [comment]
Did anyone solve this issue?

**Comment by mattBrighto** [comment]
try this
`// Include tasks without date
var hasDate = taskText.match(/(\d{4}-\d{2}-\d{2})/);
if (!Object.values(happens).length && !hasDate) {
	tasks[i].order = taskOrder.indexOf("unplanned");
	timelineDates.push(today);
	happens["unplanned"] = today;
}`
